tests/Unit/ParallelAlgorithms/Events/Test_ObserveNorms.cpp

```diff
@@ -274,16 +273,23 @@ void test(const std::unique_ptr<ObserveEvent> observe,
   CHECK(results.l2_norm_values[3] == approx(95.3187634554008838));
 
   // Check L2 integral norms
-  if (basis != Spectral::Basis::FiniteDifference) {
-    CHECK(results.reduction_names[15] == "L2IntegralNorm(Var1)");
-    CHECK(results.reduction_names[16] == "L2IntegralNorm(Var1_x)");
-    CHECK(results.reduction_names[17] == "L2IntegralNorm(Var1_y)");
-    CHECK(results.reduction_names[18] == "L2IntegralNorm(Var1_z)");
+  CHECK(results.reduction_names[15] == "L2IntegralNorm(Var1)");
+  CHECK(results.reduction_names[16] == "L2IntegralNorm(Var1_x)");
+  CHECK(results.reduction_names[17] == "L2IntegralNorm(Var1_y)");
+  CHECK(results.reduction_names[18] == "L2IntegralNorm(Var1_z)");
+  if (basis != Spectral::Basis::FiniteDifference){
     CHECK(results.l2_integral_norm_values[0] == approx(124.18131904598212145));
     CHECK(results.l2_integral_norm_values[1] == approx(41.36826480931165406));
     CHECK(results.l2_integral_norm_values[2] == approx(68.22267462752640199));
     CHECK(results.l2_integral_norm_values[3] == approx(95.15951520123110186));
   }
+  else{
```

Comment:
could you clang-format this block please? The for and else brace spacing specifically, also a space before `{` on line 280